config.py

```diff
@@ -465,24 +465,44 @@
 # XXX not needed if bids events are present
 # min_event_duration = 0.002
 
-#  `event_id`` : dict
+#  ``event_id`` : dict
 #    Dictionary that maps events (trigger/marker values)
-#    to conditions.
+#    to conditions. The ``/``` character can be used to group conditions.
+#    See the "Subselecting epochs" tutorial for more information:
+#    https://mne.tools/stable/auto_tutorials/epochs/plot_10_epochs_overview.html#subselecting-epochs  # noqa: 501
 #
 # Example
 # ~~~~~~~
+# Do not allow for easy grouping of conditions: left and right auditory
+# stimulation are treated entirely independently:
+# >>> event_id = {'auditory_left': 1, 'auditory_right': 2}`
+#
+# Allow for grouping of all ``auditory`` conditions, regardless of
+# stimulation side:
 # >>> event_id = {'auditory/left': 1, 'auditory/right': 2}`
-# or
-# >>> event_id = {'Onset': 4} with conditions = ['Onset']
+#
+# Allow for grouping into ``auditory``, ``visual``, ``left``, and ``right``:
+# >>> event_id = {'auditory/left': 1, 'auditory/right': 2,
+#                 'visual/left': 3, 'visual/right': 4}`
 
 # event_id = {'auditory/left': 1, 'auditory/right': 2}
 
 #  `conditions`` : dict
-#    List of condition names to consider. Must match the keys
-#    in event_id.
+#    List of condition names to consider. This can either be the keys of
+#    ``event_id``, or – if event names were specified with ``/`` for
+#    grouping – the name of the *grouped* condition (i.e., the
+#    condition name before or after that ``/`` that is shared between the
+#    respective conditions you wish to group). See the "Subselecting epochs"
+#    tutorial for more information: https://mne.tools/stable/auto_tutorials/epochs/plot_10_epochs_overview.html#subselecting-epochs  # noqa: 501
```

Comment:
Do you explain what `None` does?

Reply:
No, it was not documented before and I have no idea what it does :D Can you enlighten me? Then I'd be happy to add it to the description!

Reply:
I think `None` will consider all epochs instead of subselecting some

Reply:
Wow that’s quite a stretch then, semantically... None means all. We should reconsider this (in a separate PR)

Reply:
So I've checked the code and it appears to me that `conditions = [None]` shouldn't really work. So for now, I have removed it from the example `config.py`: 25ce6a0